### PR TITLE
api show list post

### PR DESCRIPTION
**Pull request by ghost**
## Description
write api show list post
## Reference Requirements
- Task ID: TI-1

## Migrations
 NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

File: app/Http/Controllers/Api/ProductController.php [hunk 1]
```diff
@@ -33,4 +34,15 @@ public function index(SortApiProductRequest $request)
...
+
+    /**
+     * Get all product's post
+     *
+     * @return Illuminate\Http\Response
+     */
+    public function getPosts()
+    {
+        $posts = Post::with('product', 'user.userInfo')->paginate(config('define.number_page_posts_user'));
+        return $this->responsePaginate($posts);
```
Comment (at-dungle) [inline app/Http/Controllers/Api/ProductController.php]: có cần truyền tham số 200 trong hàm responsePaginate

Comment (at-thaitran) [reply app/Http/Controllers/Api/ProductController.php]: không

Comment (at-dungle) [reply app/Http/Controllers/Api/ProductController.php]: v thì ok

**Comment by at-dungle** [comment]
V thì ok

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**